mypy.ini

```diff
@@ -0,0 +1,16 @@
+[mypy]
+packages = elementary
+exclude = ^elementary/monitor/dbt_project/
+allow_redefinition = True
+
+[mypy-alive_progress]
+ignore_missing_imports = True
```

Comment:
For every library that didn't have a "stubs" library I needed to do this (See the libs I added to `dev-requirements.txt`)